docs/en/solutions/Best_Practices_for_Using_Pipelines_in_AI_Scenarios.md

```
@@ -0,0 +1,493 @@
+    value: $(body.project.path_with_namespace)
+  ......
+```
+
+- **Interceptor / ClusterInterceptor**: An interceptor responsible for filtering and judging events to prevent illegal or unwanted events from triggering pipeline execution
+- **Trigger**: Responsible for associating events, interceptors, and pipelines
+
+After understanding the above, define custom events and use Tekton's Trigger mechanism to handle the relationship between pipelines.
+
+Taking model training as an example:
+
+**Step 1: Define the Model Training Completion Event**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)